src/pages/vault/tutorials/[...tutorialSlug]/index.tsx

```diff
@@ -46,5 +45,4 @@ export async function getStaticPaths(): Promise<
   }
 }
 
-VaultTutorialPage.layout = CoreDevDotLayout
```

Comment:
The underlying `TutorialView` uses `SidebarSidecarLayout`, which uses `BaseNewLayout`, which uses `CoreDevDotLayout`. This layout is duplicative, and was causing `ConsentManager` to take on the dev-dot rather than marketing `font-family`.